modules/client-configuration/pages/autoinst-profiles.adoc

```diff
@@ -1,32 +1,51 @@
 
-To see and manage your autoinstallation profiles, navigate to menu:Systems[Autoinstallation > Profiles].
+The most important part of the profile is the "autoinstallation file".
+When you perform an installation manually, you must provide information to the installer, such as partitioning and networking information and user details.
+The autoinstallation file is a method of providing this information in a scripted form.
+This type of file is sometimes also referred to as an "answers file".
 
+Within {productname}, you can use two different types of profiles, depending on the operating system of the clients you want to install:
 
+* For {sle} clients, use {ay}.
+* For {rhel} clients, use {kickstart}.
 
-== Create an Autoinstallation Profile
+You can use both {ay} and {kickstart} profiles if you want to install clients with different operating systems.
 
-You can create your autoinstallation profile directly in the {productname} {webui}.
-The simplest way to create a profile is to upload an {ay} or {kickstart} file.
-Alternatively, for {kickstart} only, you can use the {webui} wizard.
+For information about how to declare profiles, see xref:client-configuration:autoinst-profiles.adoc#declare-profile[Declare the Profile]
+
+For information about {ay} profiles, see xref:client-configuration:autoinst-profiles.adoc#autoyast[AutoYast Profiles].
+
+For information about {kickstart} profiles, see xref:client-configuration:autoinst-profiles.adoc#kickstart[Kickstart Profiles].
 
-Before you begin, you must have created an autoinstallation distribution, and prepared an autoinstallation file.
-For more information about autoinstallation distributions and files, see xref:client-configuration:autoinst-setup.adoc[].
+The autoinstallation file contained in the profile can include variables and code snippets.
+For information about variables and code snippets, see xref:client-configuration:autoinst-profiles.adoc#templates-syntax[Templates Syntax].
+
+
+[[declare-profile]]
+== Declare the Profile
+
+When you have prepared an autoinstallation file and distribution, you can create profiles to manage autoinstallation on your {productname} Server.
+For a given distribution, the profile will determine how to install this distribution.
+
+One way to create a profile is to upload an {ay} or {kickstart} file.
+Alternatively, for {kickstart} only, you can use the {webui} wizard.
 
 
 
 .Procedure: Creating an Autoinstallation Profile by Upload
 . In the {productname} {webui}, navigate to menu:Systems[Autoinstallation > Profiles].
 . Click btn:[Upload Kickstart/Autoyast File].
 . In the [guimenu]``Label`` field, type a name for the profile.
-    Do not use spaces.
+  Do not use spaces.
 . In the [guimenu]``Autoinstall Tree`` field, select the autoinstallation distribution to use for this profile.
 . In the [guimenu]``Virtualization Type`` field, select the type of virtualization to use for this profile, or select ``None`` for no virtualization.
```

Comment:
```suggestion
. In the [guimenu]``Virtualization Type`` field, select the type of virtualization to use for this profile, or select ``None`` if you do not want to use this profile to create a new virtual machine.
```

Reason: Re-install of a virtual machine must use a "normal" profile. The extra values asked are only useful to create a  VM but not for any other reason.

Reply:
ok.